tasks/mozfest_backups/utilities.py

```diff
@@ -0,0 +1,341 @@
+
+    :param file_list: List of all files on S3
+    :param guidebook_resource: Guidebook resource type. Can be "guides", "sessions", "schedule-tracks" or "locations"
+    :return: A list of files filtered by the Guidebook resource type passed in param
+    """
+
+    regex = re.compile(guidebook_resource)
+    filtered_list = [file for file in file_list if regex.match(file["Key"])]
+
+    return filtered_list
+
+
+# Check metadata of the latest uploaded file for each type and alert if older than 3 hours
+def is_stale(file_list):
+    """
+    Check if the file was uploaded less than 3 hours ago. If not, posted an alert in #mofo-mozfest-backup slack
+    channel.
+
+    :param file_list: List of backup files
+    :return:
+    """
+
+    latest_file = max(file_list, key=lambda o: o["LastModified"])
+
+    time_diff = get_time_diff(latest_file["LastModified"])
+
+    if time_diff >= timedelta(hours=3):
+        payload = {
+            "message_type": "CRITICAL",
+            "entity_id": "MozfestBackupScript",
+            "entity_display_name": f"Mozfest backup task failed: {latest_file['Key']} was not updated for {time_diff}",
+            "state_message": f"The scheduled task in charge of Mozfest Guidebook backups failed: {latest_file['Key']} "
+            f"was not updated for {time_diff}."
+            f"This task is running on the Heroku app 'mofo-cron' and is a 'clock' process."
+            f"Backups are uploaded to S3, in the '{S3_BUCKET}' bucket in mofo-project."
+            f"Logs are available on Logentries: "
+            f"https://eu.logentries.com/app/3aae5f3f#/search/log/628eb861?last=Last%2020%20Minutes",
+        }
+        requests.post(
+            f"https://alert.victorops.com/integrations/generic/20131114/alert/{VICTOROPS_KEY}",
+            json=payload,
+        )
+        print(
+            f"Failure: the file '{latest_file['Key']}' was not updated for {time_diff}. An alert has been sent."
+        )
+
+
+def delete_old_backups(file_list):
+    """
+    Delete backups that are older than 48h.
+
+    :param file_list: List of backups on S3
+    :return:
+    """
+
+    print("Deleting files that are older than 48h")
+
+    for file in file_list:
+        time_diff = get_time_diff(file["LastModified"])
+        if time_diff >= timedelta(days=2):
+            print(f"Deleting {file['Key']}")
+            s3.delete_object(Bucket=S3_BUCKET, Key=file["Key"])
+
+
+def get_guidebook_content(guidebook_resource):
+    """
+    Get content (sessions, tracks, etc) from Guidebook.
+
+    :param guidebook_resource: Guidebook resources type. Can be "guides", "sessions", "schedule-tracks" or "locations"
+    :return: Content from guidebook (dict)
+    """
+
+    # Guides don't have 'next' or 'results' keys
+    if guidebook_resource == "guides":
+        resource_url = API_URL + guidebook_resource + f"/{GUIDE_ID}"
+        r = requests.get(resource_url, headers={"Authorization": "JWT " + API_KEY})
```

Comment:
Consider extracting the headers kwarg and define it once to minimize repetition:

``` python
request_wkargs = {
    'headers': {'Authorization': f'JWT {API_KEY}'}
}

# then everywhere you need it:
r = requests.get(resource_url, **request_kwargs)
```